lib/strip_js.ex

```diff
@@ -100,9 +98,9 @@ defmodule StripJs do
       ""
 
       iex> StripJs.clean_html("&lt;script&gt; console.log('oh heck'); &lt;/script&gt;")
-      "&lt;script&gt; console.log('oh heck'); &lt;/script&gt;"  ## HTML entity attack didn't work
+      "&lt;script&gt; console.log(&apos;oh heck&apos;); &lt;/script&gt;"  ## HTML entity attack didn't work
```

Comment:
is this actually desired behavior?

Reply:
you know, I am unsure

Reply:
it's possible we don't want `'` as part of the regex I added to Floki. I can circle the wagons on that

Reply:
actually, my bad, that's not a `Floki` thing. That's all on me pulling in the `HtmlEntities.encode()` functionality, which I'm going to back out cuz it might cause unexpected behavior changes that I'd like to avoid for now.